development.bats

```diff
@@ -132,3 +132,38 @@ END
     run testCommit $REPO_PATH
     [ ${status} -eq 0 ]
 }
+
+@test "it allows 23.22.13.113 as an ip address" {
+    cat > $REPO_PATH/ip.txt <<END
+GSA IP address is 23.22.13.113
+END
+    run testCommit $REPO_PATH
+    [ ${status} -eq 0 ]
+}
+
+@test "it allows feed-a-bee as an example secret hex value" {
+    cat > $REPO_PATH/sample.yml <<END
+enc_key: feedabee
```

Comment:
Are the dashes in the test description just to make it easier to read for a human?  I just want to make sure this isn't a discrepancy. :-)

Reply:
Yes, readability and memorability. I used to use "deadbeef" for these examples, but "feedabee" is less gruesome.